src/Aspire.Hosting.Azure/AzureBlobStorageResource.cs

```diff
@@ -23,22 +23,8 @@ public class AzureBlobStorageResource(string name, AzureStorageResource storage)
     /// <summary>
     /// Gets the connection string template for the manifest for the Azure Blob Storage resource.
     /// </summary>
-    public string ConnectionStringExpression => Parent.BlobEndpoint.ValueExpression;
-
-    /// <summary>
-    /// Gets the connection string for the Azure Blob Storage resource.
-    /// </summary>
-    /// <param name="cancellationToken"> A <see cref="CancellationToken"/> to observe while waiting for the task to complete.</param>
-    /// <returns>The connection string for the Azure Blob Storage resource.</returns>
-    public async ValueTask<string?> GetConnectionStringAsync(CancellationToken cancellationToken = default)
-    {
-        if (Parent.ProvisioningTaskCompletionSource is not null)
-        {
-            await Parent.ProvisioningTaskCompletionSource.Task.WaitAsync(cancellationToken).ConfigureAwait(false);
-        }
```

Comment:
I assume this is no longer needed because a BicepOutputReference will await the provisioning task when someone asks for it's GetValueAsync?

Reply:
Correct.